CONTRIBUTING.md

```diff
@@ -31,9 +31,22 @@ To contribute to feature development or bug fixes. Check our [issue tracker](htt
 
 #### Setup Simulator manually
 
-##### Window One
+##### Pre-commit
+
+We use pre-commit to run some checks before you commit your code. It should get installed by the next steps.
+
+###### Frontend
```

Comment:
@AgustinRamiroDiaz, can we also add the rules that any contributor should meet somewhere?

Reply:
@cristiam86 what exactly do you mean by this? Pointing them out to the `pre-commit.yml` file? Or explaining the rules one by one here?

Reply:
I would just list the rules and a couple of works that could describe them.

Reply:
Great! I've documented them below :D